sdformat_mjcf_utils/sdformat_mjcf_utils/sdf_utils.py

```diff
@@ -31,6 +31,32 @@ def vec3d_to_list(vec):
     return [vec.x(), vec.y(), vec.z()]
 
 
+def list_to_vec3d(list):
+    """Convert a list to a Vector3d object.
+    :param List of values of the x, y, and z components of `vec`
+    respectively.
+    :return: Vector3d object
+    :rtype: ignition.math.Vector3d
+    """
+    return Vector3d(list[0], list[1], list[2])
+
+
+def get_angle(element):
```

Comment:
I think `get_angle` is confusing. Can you rename this to `get_rotation` or `get_orientation`?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/40148462713bdbba5fa3ba236d57bf8c24596e61